cli/cmd/port-forward/port_forward.go

```diff
@@ -27,88 +31,78 @@ func init() {
 	portForwardCmd.Flags().IntP("prometheus", "p", common.LISTEN_PORT_PROM, "Port to listen from for Prometheus")
 	portForwardCmd.Flags().IntP("promscale", "c", common.LISTEN_PORT_PROMSCALE, "Port to listen from for the Promscale")
 	portForwardCmd.Flags().IntP("promlens", "l", common.LISTEN_PORT_PROMLENS, "Port to listen from for PromLens")
+	portForwardCmd.Flags().IntP("jaeger", "j", common.LISTEN_PORT_JAEGER, "Port to listen from for Jaeger")
 }
 
 func portForward(cmd *cobra.Command, args []string) error {
 	var err error
 
-	var timescaledb int
-	timescaledb, err = cmd.Flags().GetInt("timescaledb")
+	timescaledbPort, err := cmd.Flags().GetInt("timescaledb")
 	if err != nil {
 		return fmt.Errorf("could not port-forward: %w", err)
 	}
 
-	var grafana int
-	grafana, err = cmd.Flags().GetInt("grafana")
+	grafanaPort, err := cmd.Flags().GetInt("grafana")
 	if err != nil {
 		return fmt.Errorf("could not port-forward: %w", err)
 	}
 
-	var prometheus int
-	prometheus, err = cmd.Flags().GetInt("prometheus")
+	prometheusPort, err := cmd.Flags().GetInt("prometheus")
 	if err != nil {
 		return fmt.Errorf("could not port-forward: %w", err)
 	}
 
-	var promscalePort int
-	promscalePort, err = cmd.Flags().GetInt("promscale")
+	promscalePort, err := cmd.Flags().GetInt("promscale")
 	if err != nil {
 		return fmt.Errorf("could not port-forward: %w", err)
 	}
 
-	var promlensPort int
-	promlensPort, err = cmd.Flags().GetInt("promlens")
+	promlensPort, err := cmd.Flags().GetInt("promlens")
+	if err != nil {
+		return fmt.Errorf("could not port-forward: %w", err)
+	}
+
+	jaegerPort, err := cmd.Flags().GetInt("jaeger")
 	if err != nil {
 		return fmt.Errorf("could not port-forward: %w", err)
 	}
 
-	k8sClient := k8s.NewClient()
 	// Port-forward TimescaleDB
 	// if db-uri exists skip the port-forwarding as it isn't the db within the cluster
+	k8sClient := k8s.NewClient()
 	uri, err := utils.GetTimescaleDBURI(k8sClient, root.Namespace, root.HelmReleaseName)
 	if err != nil {
 		return err
 	}
 	if uri == "" {
-		podName, err := k8sClient.KubeGetPodName(root.Namespace, map[string]string{"release": root.HelmReleaseName, "role": "master"})
-		if err != nil {
-			return fmt.Errorf("could not port-forward: %w", err)
-		}
-
-		_, err = k8sClient.KubePortForwardPod(root.Namespace, podName, timescaledb, common.FORWARD_PORT_TSDB)
-		if err != nil {
-			return fmt.Errorf("could not port-forward: %w", err)
+		if err := timescaledb.PortForwardTimescaleDB(timescaledbPort); err != nil {
```

Comment:
IMHO it would be better to have the same api for all packages when it comes to PortForward command. It might be less error prone to have:
```
timescaledb.PortForward(timescaledbPort);
grafana.PortForward(grafanaPort);
...
```
instead of
```
timescaledb.PortForwardTimescaleDB(timescaledbPort);
grafana.PortForwardGrafana(grafanaPort)
```